src/prime_testd.cpp

```diff
@@ -0,0 +1,102 @@
+#include <functional>
```

Comment:
can you maybe explain what this new binary is needed for, can you not test this from inside prime_httpd?